### PR TITLE
DataPrep Table

### PR DESCRIPTION
**Pull request by elfenheart**
### Feature
- User should be able to see the data table

#### Note
- Space at the bottom of the table is placeholder for the CLI
- Currently branched off from `feature/ui-data-prep-top-panel` for ease of review. Will change branch to `develop` once the other branches are merged.

### CODE REVIEW

File: cdap-ui/app/cdap/components/DataPrep/DataPrep.scss [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+.dataprep-container {
+  &.error { padding-top: 25px; }
+
+  .dataprep-body.row {
+    // height calculations
+    // 100vh - height of navbar - top panel - dataprep cli - footer
+    // 100vh - 50px - 50px - 50px - 54px
+    margin-right: 0;
+    margin-left: 0;
+    height: calc(100vh - 154px);
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/DataPrep/DataPrep.scss]: Based on the above comment this should be `100vh - 204px` right? Am I missing something here?

**Review by ajainarayanan — APPROVED** [review]
Minor comment. Otherwise simple change 👍 LGTM

File: cdap-ui/app/cdap/components/DataPrep/DataPrep.scss (hunk 1 above)
Comment (elfenheart) [reply cdap-ui/app/cdap/components/DataPrep/DataPrep.scss]: my bad, forgot to update comment. The CLI is inside this, so didn't need to subtract the CLI height anymore

**Review by ajainarayanan — APPROVED** [review]

**Comment by elfenheart** [comment]
Closing in favor of https://github.com/caskdata/cdap/pull/8310

**Pull request closed without merging**